modules/claims_api/app/controllers/claims_api/v1/claims_controller.rb

```diff
@@ -64,8 +64,19 @@ def fetch_errored(claim)
 
       def format_evss_errors(errors)
         errors.map do |error|
-          formatted = error['key'] ? error['key'].gsub('.', '/') : error['key']
-          { status: 422, detail: "#{error['severity']} #{error['detail'] || error['text']}".squish, source: formatted }
+          if error&.dig('text')
+            formatted = error['key']
+            { status: 422, detail: "#{error['severity']} || #{error['text']}".squish,
+              source: formatted }
+          elsif error&.dig('text', 'messages')
+            formatted = error['text']['messages'][0]['key']
+            { status: 422, detail: "#{error['severity']} || #{error['text']['messages'][0]['text']}".squish,
```

Comment:
I am also unsure if we can add the ' || ' in the error message return, but I guess that is kind of a question for someone else to validate, it just seems like a change in the response to me, if we are allowed to change that then this one can just be resolved.